lib/statesman/adapters/active_record.rb

```diff
@@ -197,8 +197,8 @@ def with_updated_timestamp(params)
     end
 
     class ActiveRecordAfterCommitWrap
-      def initialize
-        @callback = Proc.new
+      def initialize(&block)
+        @callback = Proc.new(&block)
```

Comment:
Could this not just be 
```suggestion
        @callback = block
```
?